revolv/administrator/views.py

```diff
@@ -29,6 +35,423 @@ def get_context_data(self, **kwargs):
+        net_cash_in: Net inflow of cash during this period.
+        net_cash_out: Net outflow of cash during this period.
+    """
+    def setTimePeriodAdjustments(self, filtered_dict, date, temp_start_date, temp_end_date):
+        filtered_dict['cash_in']['adjustments'][date]['transactions'] = {}
+        current_dict = filtered_dict['cash_in']['adjustments'][date]['transactions']
+        for adjustment in AdminAdjustment.objects.adjustments(start_date=temp_start_date, end_date=temp_end_date, cash_type='cash_in'):
+            if adjustment.name in current_dict:
+                current_dict[adjustment.name] += adjustment.amount
+            else:
+                current_dict[adjustment.name] = adjustment.amount
+        
+        current_dict = filtered_dict['cash_in']['adjustments'][date]
+        total = sum([adjustment.amount for adjustment in AdminAdjustment.objects.adjustments(start_date=temp_start_date, end_date=temp_end_date, cash_type='cash_in')])
+        current_dict['total'] = total
+
+        filtered_dict['cash_out']['adjustments'][date]['transactions'] = {}
+        current_dict = filtered_dict['cash_out']['adjustments'][date]['transactions']
+        for adjustment in AdminAdjustment.objects.adjustments(start_date=temp_start_date, end_date=temp_end_date, cash_type='cash_out'):
+            if adjustment.name in current_dict:
+                current_dict[adjustment.name] += adjustment.amount
+            else:
+                current_dict[adjustment.name] = adjustment.amount
+
+        current_dict = filtered_dict['cash_out']['adjustments'][date]
+        total = sum([adjustment.amount for adjustment in AdminAdjustment.objects.adjustments(start_date=temp_start_date, end_date=temp_end_date, cash_type='cash_out')])
+        current_dict['total'] = total
+    
+    """
+    Creates and fills in the 'repayments' subdictionary of 'cash_in' and 'cash_out'
+    within the JSON serializable dictionary for a certain time period and project.
+
+    :args:
+        filtered_dict: The JSON serializable dictionary.
+        date: A string represenation of temp_start_date.
+        proj: The project of interest.
+        temp_start_date: The start date of this time period.
+        temp_end_date: The end date of this time period.
+        revolv_earnings_rate: A float from 0-1 representing what percentage of repayments RE-volv takes as earnings.
+    """
+    def setRepaymentTableValues(self, filtered_dict, date, proj, temp_start_date, temp_end_date, revolv_earnings_rate):
+        current_dict = filtered_dict['cash_in']['repayments'][date][proj.title]
+        total = sum([repayment.amount for repayment in AdminRepayment.objects.repayments(project=proj, start_date=temp_start_date, end_date=temp_end_date)])
+        current_dict['total'] = total
+        current_dict['revolv_earnings'] = total * revolv_earnings_rate
+        current_dict['retained_ROI'] = total * (1 - revolv_earnings_rate)
```

Comment:
ROI is actually 'return on investment' so I think it should be okay! I didn't want to put retained earnings because I think that's easily confused with revolv_earnings, but i'm not sure

do you think that would be more clear?